MaterialLib/MPL/Properties/CapillaryPressureSaturation/CapillaryPressureVanGenuchten.cpp

```diff
@@ -75,6 +75,8 @@ PropertyDataType CapillaryPressureVanGenuchten::dValue(
     const double val1 = std::pow(Se, -1.0 / _m);
     const double val2 = std::pow(val1 - 1.0, -_m);
     return _p_b * (_m - 1.0) * val1 * val2 / (_m * (S - _residual_saturation));
+    */
+    return 0;
 }
 
 PropertyDataType CapillaryPressureVanGenuchten::d2Value(
```

Comment:
Now I think this member, d2Value, can be removed. It was added purely for the test purpose and it will not be used anywhere in  computation.